changelogs/unreleased/3641-arush-sal

```diff
@@ -0,0 +1 @@
+Add Label to BackupSpec so that labels can explicitly be provided to Schedule.Spec.Template.Metadata.Labels which will be reflected on the backups created.
```

Comment:
Please document how the existing behavior is being changed by this.

Reply:
I've added docs here: https://github.com/vmware-tanzu/velero/pull/3641/files#diff-a521c2750535d66cd8cf8c68f1ee1e021dcecfd852b40de729c2cbc9c776b4d5 any where else you feel this should be documented?

Reply:
Those are good, thanks